### ISSUE TITLE
[Windows] Dev branch, couldn't find provider vodo

### ISSUE THREAD

**Issue by exadeci** [popcorntime/popcorntime#741]
Operating System Version:
Windows 10

Popcorn Time Version:
f3135466bd5b103c202929e21c45e7b2db25b1ae

#### Expected Behaviour
Starts and loads content

#### Actual Behaviour
The interface stays blank

#### Steps to reproduce the behaviour
1. Setup development branch on windows
2. Build it 
3. Launch it

The error from the console:
```
app.js:21 [ERROR] class Vodo extends Provider {
    constructor (args, config = defaultConfig) {
        super(args, config)

        this.apiUrl = this.args.urlList
    }

    updateAPI() {
        var defer = Q.defer();
        debug('Request to Vodo', this.apiUrl);
        axios(this.apiUrl[0], {
            strictSSL: false,
            json: true,
            timeout: 10000
        })
            .then((res) => {
                let data = res.data
                /*
                   data = _.map (helpers.formatForButter(data), (item) => {
                   item.rating = item.rating.percentage * Math.log(item.rating.votes);
                   return item;
                   });
                 */
                db.insert(formatForButter(data.downloads), (err, newDocs) => {
                    if (err) {
                        debug('Vodo.updateAPI(): Error inserting', err);
                    }

                    defer.resolve(newDocs);
                });
            })

        return defer.promise;
    }
    fetch(filters = {}) {
        if (!this.fetchPromise) {
            this.fetchPromise = this.updateAPI();
        }

        var defer = Q.defer();
        var params = {
            sort: 'rating',
            limit: 50
        };
        var findOpts = {};

        if (filters.keywords) {
            findOpts = {
                title: new RegExp(filters.keywords.replace(/\s/g, '\\s+'), 'gi')
            };
        }

        if (filters.genre) {
            params.genre = filters.genre;
        }

        if (filters.order) {
            params.order = filters.order;
        }

        if (filters.sorter && filters.sorter !== 'popularity') {
            params.sort = filters.sorter;
        }

        var sortOpts = {};
        sortOpts[params.sort] = params.order;

        this.fetchPromise.then(() => {
            db.find(findOpts)
              .sort(sortOpts)
              .skip((filters.page - 1) * params.limit)
              .limit(Number(params.limit))
              .exec((err, docs) => {
                  docs.forEach((entry) => {
                      entry.type = 'movie';
                  });

                  return defer.resolve({
                      results: docs,
                      hasMore: docs.length ? true : false
                  });
              });
        });

        return defer.promise;
    }

    detail (torrent_id, old_data) {
        return Q(old_data);
    }
} undefined doesnt have a namewin.error @ app.js:21
app.js:21 [ERROR] couldn't find provider vodowin.error @ app.js:21
app.js:6 [DEBUG] Not updating because we are running the latest version

```

**Comment by Persei08** [comment]
Seem related to last updates from [butter-provider-vodo github repo](https://github.com/butterproviders/butter-provider-vodo)

Quick fix:
edit `package.json` (line 47) and replace:
`"butter-provider-vodo": "git+https://github.com/butterproviders/butter-provider-vodo",`
by 
`"butter-provider-vodo": "0.3.1",`

then: 
- delete `node_modules/` folder.
- delete package-lock.json,
- re-exec `npm install`, `gulp build`, `gulp run`


edit: add step to delete package-lock.json after @exadeci feedback

**Comment by exadeci** [comment]
@Persei08  Great! Thanks for the detailed fix, to add to that I had to delete `package-lock.json` but I guess a `npm update` would have worked also.

**Comment by Persei08** [comment]
@exadeci : I 'm not sure, I tried `npm update` but it don't updated butter-provider-vodo, maybe because the git version is also 0.3.1 or because I've done it wrong. I'm not very familiar with npm.

thx for feedback

**Comment by exadeci** [comment]
You and me both :D 
But I noticed that it was still displaying the `git+` in version instead of `0.3.1`

**Comment by ChrisAlderson** [comment]
Fixed with #771
